lib/adapters/leveldb/index.js

```diff
@@ -10,7 +10,16 @@ function requireLeveldown() {
   // de-optimalization
   try {
     originalLeveldown = require('leveldown');
-  } catch (e) {}
+  } catch (err) {
+      if (err && err.message && err.message.match('Module version mismatch')) {
+          throw new Error([
+            err.message,
+            'This generally implies that leveldown was built with a different',
+            'version of node than that which is running now.  You may try',
+            'fully removing and reinstalling PouchDB or leveldown to resolve.'
+          ].join(' '));
+      }
```

Comment:
Thanks Nolan!

> but if we throw within a catch, then that removes the whole value of the try/catch

my take was that that this was a valid case to throw, as I assumed that the "Cannot find module" error was the _actual error that this block was intended to catch_.  if this were the case, in retrospect, it may have sniffed more specifically for that error.  however, i'm betting that it was an oversight [when it was implemented](https://github.com/pouchdb/pouchdb/commit/da09b3e6fcf3d4cdf04880ce9433d8e47600b36f).

I'm totally down with the `console.warn` suggestion, but i'm now thinking that the actual ideal solution would be to:

1. test if `opts.db` is passed, then on falsy,
1. try to import leveldown
1. sniff for the module not found error
1. if module is not found, we should throw the `"leveldown not available "` error per the status-quo, otherwise,
1. throw the actual nodejs produced error, unsquashed

right now, importing leveldown in the try catch doesn't offer much value as the next executing control block just throws the "not available" error (in probably ~99% of use cases).  let me know your thoughts!